pytest_pylint/tests/test_pytest_pylint.py

```diff
@@ -142,6 +142,7 @@ def test_pylintrc_file_beside_ini(testdir):
     assert 'Line too long (10/3)' in result.stdout.str()
 
 
+# pylint: disable=not-callable
```

Comment:
This disables the message at the module scope (whole file). It needs to move down into the function for the proper scoping